resources/fetching/factory/eks_factory.go

```diff
@@ -27,7 +27,8 @@ import (
 
 	"github.com/elastic/cloudbeat/resources/fetching"
 	"github.com/elastic/cloudbeat/resources/fetching/condition"
-	"github.com/elastic/cloudbeat/resources/fetching/fetchers"
+	awsf "github.com/elastic/cloudbeat/resources/fetching/fetchers/aws"
+	k8sf "github.com/elastic/cloudbeat/resources/fetching/fetchers/k8s"
```

Comment:
abbreviations are usually something we avoid, but if you think it's clear I'm fine with it